applications/machine_learning/README.rst

```diff
@@ -79,6 +79,31 @@ By default, the application uses pretrained machine leaning models deployed in `
 The application displays LED effects that correspond to the machine learning results.
 For more detailed information, see the `User interface`_ section.
 
+Power management
+================
+
+Reducing power consumption is important for every battery-powered device.
+
+In the nRF Machine Learning application, you can suspend or turn off modules if they are not in use.
```

Comment:
That's partially true - we only set a global power state for whole application, we do not have per-module power state.

Reply:
Maybe:
`In the nRF Machine Learning application, application modules are automatically suspended or turned off if the device is not in use for predefined period of time.`?